projects/clr/hipamd/packaging/CMakeLists.txt

```diff
@@ -35,9 +37,9 @@ set(CPACK_DEB_COMPONENT_INSTALL ON)
 
 ###Set License####
 set(CPACK_RESOURCE_FILE_LICENSE ${hip_SOURCE_DIR}/LICENSE.md)
-install(FILES ${CPACK_RESOURCE_FILE_LICENSE} DESTINATION ${CMAKE_INSTALL_DOCDIR} COMPONENT binary)
+install(FILES ${CPACK_RESOURCE_FILE_LICENSE} DESTINATION ${CMAKE_INSTALL_DATADIR}/doc/${PROJECT_NAME} COMPONENT binary)
```

Comment:
I'm curious why we're changing to use CMAKE_INSTALL_DATADIR instead of CMAKE_INSTALL_DOCDIR?  I'm fine with it, just curious.

Reply:
> I'm curious why we're changing to use CMAKE_INSTALL_DATADIR instead of CMAKE_INSTALL_DOCDIR? I'm fine with it, just curious.

Here, the project setting in the projects/clr/opencl/CMakeLists.txt (as **opencl**) is passed on to subdirectory packaging, and CMAKE Flags are not redefined at subdirectory level in current code, but since project name at subdirectory is **rocm-opencl**, So proposing to use this by explicit override. (with PROJECT_NAME defined at subdir level) as it required folder name, since it is the package name. Hope it clarifies.